src/containers/CryptoList.tsx

```diff
@@ -0,0 +1,79 @@
+import List from "../components/List";
+import CoinData from "../types/CoinData";
+import React from "react";
+import { FlatList, Image, Text, TouchableOpacity, View } from "react-native";
+import styled from "styled-components";
+
+const cryptos: CoinData[] = [
+	{
+		id: "1",
+		name: "Bitcoin",
+		symbol: "BTC",
+		price: 7215.68,
+		percent: 1.82,
+		img: require("../assets/img/bitcoin.png"),
+	},
+	{
+		id: "2",
+		name: "Ethereum",
+		symbol: "ETH",
+		price: 146.83,
+		percent: 1.46,
+		img: require("../assets/img/ethereum.png"),
+	},
+	{
+		id: "3",
+		name: "XRP",
+		symbol: "XRP",
+		price: 0.220568,
+		percent: -2.47,
+		img: require("../assets/img/xrp.png"),
+	},
+];
+
+const TopBar = styled(View)`
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 24px;
+  padding-top: 40px;
+  height: 130px;
```

Comment:
if you are alredy using a padding, you shoul use it to set the height of the topbar, and it will be responsive to the size of the content